docs/understanding-tracking-design/managing-your-tracking-scenarios/ui/index.md

```diff
@@ -2,6 +2,9 @@
 title: "Managing tracking scenarios in the Console"
 sidebar_label: "Using the UI 🆕"
 sidebar_position: 1
+sidebar_custom_props:
+  offerings:
+    - enterprise
```

Comment:
I thought it was the opposite — UI on enterprise and cloud, and API enterprise only?